docs/samples/box/canvas.md

```diff
@@ -0,0 +1,93 @@
+# Using canvas as labels
+
+```js chart-editor
+// <block:setup:3>
+const DATA_COUNT = 12;
+const MIN = 10;
+const MAX = 100;
+
+const numberCfg = {count: DATA_COUNT, min: MIN, max: MAX};
+
+const data = {
+  labels: ['January', 'February', 'March', 'April', 'May', 'June', 'July', 'August', 'September', 'October', 'November', 'December'],
+  datasets: [{
+    data: Utils.numbers(numberCfg)
+  }]
+};
+// </block:setup>
+
+// <block:annotation1:1>
+const annotation1 = {
+  type: 'box',
+  backgroundColor: 'rgba(165, 214, 167, 0.2)',
+  borderWidth: 0,
+  xMax: 2,
+  xMin: 5,
+  label: {
+    enabled: true,
+    content: Utils.getSpiral,
```

Comment:
```suggestion
    content: Utils.getSpiral(),
```

While both syntaxes work I think this is a bit nicer looking, colors in the console look better and its a bit more clear its a function that gets called that can return something